sample/test-sample/frontend/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@eclipse-sirius/sirius-web",
+  "author": "Eclipse Sirius",
+  "version": "0.1.0",
+  "license": "EPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-sirius/sirius-web"
```

Comment:
Wrong URL, but I'm not sur this section makes sense for a sample app used for testing only.

Reply:
Done